src/Tribe/Migration/Queue.php

```diff
@@ -0,0 +1,287 @@
+<?php
+/**
+ * Queue class for migration handling.
+ *
+ * @since TBD
+ */
+
+namespace Tribe\Tickets\Migration;
+
+class Queue {
+
```

Comment:
:no_entry_sign: Missing doc comment for class Queue

<sub>hash: afe8c123d22f800cc972333caa229bde</sub>